### PR TITLE
Refactor back to old structure; Refactor annotations into separate package; Make a release

### PR DESCRIPTION
**Pull request by datakurre**
Hi @vivekkodu, 

This pull has a lot of commit, but not so much changes. In chronological order:
- I tested the package with a documentation with sub-directories and fixed (I believe) a few paths to work with documentations.
- I moved my annotation keywords into a separate package https://pypi.python.org/pypi/robotframework-selenium2screenshots
- I refactored code into the old structure
- I released a version to make able to show this to others and get feedback from the community https://pypi.python.org/pypi/sphinxcontrib-robotframework

If you create a PyPI-account later and would like to learn the packaging process and maintain the package, I can give you maintainer access for the package at PyPI.

Do you have opinions on licensing? It's now on GPL, but also MIT would be OK for me. http://stackoverflow.com/questions/3902754/mit-vs-gpl-licence

What next?

Officially, you have a week of GSOC left. You could use it for polishing the new ReST reader or add new features for the Sphinx-plugin (for example, the directive to disable robot run if all the listed screenshots have already been created). Currently, they are independent of each other and can also be separately worked on. Also, you could only learn to use these by yourself to get real benefit out of GSOC for yourself (generating screenshots automatically is a big thing for any project, I believe).

What happens after GSOC is that once the new ReST-reader is merged into robotframework, it will be maintained by Robot Framework -team with the Robot Framework user/developer community. sphinxcontrib_robotframewor, on the other hand, would be without a community at first (I can maintain and continue developing it), but I'd expect Plone-community to start using it soon and provide substantial contributions for it.

To summarize. I'm already happy with the GSOC results. There's some work left to polish things, but no more difficult issues to solve. Please, tell me, what will you be doing the rest of your GSOC and I'll try to do the rest (and try to avoid doing conflicting work).

### CODE REVIEW

**Comment by vivekkodu** [comment]
This week, i can give some time on ReST reader but would prefer to give much time to learn the real life implementations of robotframework.
For addition of features in sphinxcontrib_robotframework, that is not necessary to be included in GSOC code. So if you don't work on that now, i can work on that afterwards. But i think that should be completed before the next release of robotframework. Also i would be interested in maintainer access for the package at PyPI.

**Comment by datakurre** [comment]
@vivekkodu Go for learning more robot. I worked tonight with RestReader and updated our issue at Robot Framework googlecode tracker with new patch. I hope that it's ready now.
